source/reports/multidataset_reports/MultiDatasetReport.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+
+from source.reports.Report import Report
+
+
+class MultiDatasetReport(Report, ABC):
```

Comment:
Is it necessary to inherit ABC as well? I think it raises a warning occasionally about how it's imported (there have been some changes between python versions, not sure exactly what).

Reply:
I did it here because it was already done like this in abstract classes for encoding reports and ML reports, if you think it should be removed, it should probably be removed for all?

Reply:
I think so, do you agree? I don't even know why it is there now, since they are abstract anyway because they are not implementing abstract methods of the parent class. 🤔 🙏

Reply:
I removed it now from encodingreport, datareport, etc